clap-utils/src/input_validators.rs

```diff
@@ -287,13 +287,14 @@ where
     }
 }
 
-pub fn is_amount_or_all<T>(amount: T) -> Result<(), String>
+pub fn is_amount_or_all_or_available<T>(amount: T) -> Result<(), String>
```

Comment:
`is_amount_or_all` is used a bunch of other places that need not support `AVAILABLE`. Let's add a new method instead of renaming the existing validator. Please revert the changes to non-stake-withdraw subcommands.

Reply:
I splitted in two different functions `is_amount_or_all` & `is_amount_or_all_or_available` and reverted non-stake-withdraw subcommands as you suggested. 👍